src/components/Fields/TextField.story.js

```diff
@@ -77,13 +77,13 @@ stories.add(
 );
 
 stories.add(
-    'Label Always',
+    'Label On Hover',
     storyWrapper(
-        'Label can be forced to always appear with the hasLabelAlways prop.',
+        'Label will hide when the input does not have focus with the hasLabelOnFocus prop.',
         <TextField
             label="Always has a label"
             value="my example value"
-            hasLabelAlways
+            hasLabelOnFocus
```

Comment:
Same as with TextArea, this should be `autoHideLabel`. Please amend the story title and description too.